templates/default/push-jobs-client.rb.erb

```diff
@@ -8,13 +8,11 @@
 
 #Chef::Config.from_file(PathHelper.join(Chef::Config.platform_specific_path("/etc/chef/"), "client.rb"))
 
-node_name  `hostname`.chomp
-
-chef_server_url  "<%= @chef_server_url %>"
-
-client_key                  '/etc/chef/secure/client.pem'
-trusted_certs_dir           '/etc/chef/secure/trusted_certs'
-verify_api_cert false
+chef_server_url   '<%= @chef_server_url %>'
+node_name         '<%= @node_name %>'
+client_key        '<%= @client_key_path %>'
+trusted_certs_dir '<%= @trusted_certs_path %>'
+verify_api_cert   false
```

Comment:
I see there's a 'verify_api_cert' passed in the template definition; did you want to have <%= verify_api_cert %> here?